spec/plugins/partner_sku_reviews_spec.coffee

```diff
@@ -127,6 +130,7 @@ describe 'PartnerSkuReviews', ->
           "review": "Πριν ξεκινήσω να καταγράψω",
           "merged_review_notice_html": '',
           "rating": 5,
+          "purchased": true,
```

Comment:
If we go down the path of being more specific as mentioned in my other comment, there is no need to set this to true 
```suggestion
          "purchased": false,
```

Reply:
Same as above!